lib/default.nix

```diff
@@ -202,7 +199,7 @@ in {
   cleanGit = import ./clean-git.nix {
     inherit lib cleanSourceWith;
     git = gitMinimal;
-    inherit (pkgs.evalPackages.buildPackages) runCommand;
+    inherit (pkgs.buildPackages.buildPackages) runCommand;
```

Comment:
Are these changes actually okay? Won't this again mean that you can't eval without a build machine of the right system, because it will try and clean the source on a build machine?

Reply:
I think there is a situation when not using flakes (so `cleanGit` would have work to do) and cross compiling where this would have been useful so I have included a note in the change log.